src/components/CustomMessageSystem/index.js

```diff
@@ -13,7 +13,7 @@ const index = ({text}) => (
     }}>
     <View
       style={{
-        backgroundColor: COLORS.lightgrey,
+        backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` seems fine as long as the new color `COLORS.gray100` is defined in your `COLORS` constant and it fits with the overall theme of your application. Ensure that this color change doesn't affect the readability and accessibility of the component.

```diff
-        backgroundColor: COLORS.lightgrey,
+        backgroundColor: COLORS.gray100,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->